<launch>
	<node pkg="camera_publisher" type="camera_publisher_node" name="camera_publisher" output="screen">

	      <param name="device_name" type="string" value="/dev/video0" />
	      <param name="topic_name" type="string" value="data/video0" />

	      <param name="camera_width" type="int" value="1280" />
	      <param name="camera_height" type="int" value="720" />

	      <param name="output_width" type="int" value="100" />
	      <param name="output_height" type="int" value="100" />
	</node>
</launch>